package.json

```diff
@@ -45,20 +46,23 @@
         "babel-plugin-emotion": "^10.0.13",
         "enzyme": "^3.10.0",
         "enzyme-adapter-react-16": "^1.14.0",
+        "enzyme-to-json": "^3.3.5",
         "eslint": "^5.16.0",
         "eslint-config-react-app": "^3.0.8",
         "eslint-plugin-flowtype": "^2.50.3",
         "eslint-plugin-import": "^2.16.0",
         "eslint-plugin-jsx-a11y": "^6.2.1",
         "eslint-plugin-react": "^7.12.4",
         "jest": "^24.8.0",
+        "prop-types": "^15.7.2",
         "react": "^16.8.6",
-        "enzyme-to-json": "^3.3.5",
-        "prop-types": "^15.7.2"
+        "react-dom": "^16.8.6"
     },
     "peerDependencies": {
-        "@emotion/core": "^10.0.10",
-        "react": "^16.8.6"
+        "prop-types": "^15.7.2",
+        "@babel/runtime": "^7.11.2",
+        "react": "^16.8.6",
+        "react-dom": "^16.8.6"
```

Comment:
Does this need to be a peer dependancy? We don't directly use it in the component.

Reply:
Yes, The implementation is dom based so ideally, it should be even though there no ReactDOM.render. This makes sure you ask for the same version of react and react-dom from the library user.